.github/workflows/build.yml

```diff
@@ -36,7 +44,7 @@ jobs:
                     echo $extension_version >> $env:GITHUB_ENV
                 - name: Setup PHP
                   id: setup-php
-                  uses: php/setup-php-sdk@v0.8
+                  uses: php/setup-php-sdk@v0.10
                   with:
                       version: ${{matrix.php}}
                       arch: ${{matrix.arch}}
```

Comment:
While you're at it, consider to enable caching:
````
                      cache: true
````

Reply:
Done - thank you!